app.js

```diff
@@ -1,16 +1,19 @@
 const express = require("express");
-const { psqlErrors, customErrors, handle500s } = require("./errors/index");
+const { psqlErrors, customErrors, handle404 } = require("./errors/index.js");
 const { getTopics } = require("./controllers/topics.controllers.js");
+const { getArticleById } = require("./controllers/articles.controllers.js");
 const app = express();
 app.use(express.json());
```

Comment:
This should not be here while we are working on GET requests. This code is not needed until PATCH or POST requests.

Reply:
ok